localflavor/at/forms.py

```diff
@@ -53,8 +52,8 @@ class ATSocialSecurityNumberField(Field):
 
     def clean(self, value):
         value = super().clean(value)
-        if value in EMPTY_VALUES:
-            return ""
+        if value in self.empty_values:
+            return self.empty_value
```

Comment:
The reason I included these changes was because changing to `CharField` would introduce the option to set `empty_value` as an argument without it having any effect. `self.empty_value` defaults to an empty string and `self.empty_values` defaults to `EMPTY_VALUES` so there shouldn't be a problem with this change.

Does this reasoning sound ok to you? I'm definitely open to less invasive changes if you think that's better for a 3.1.1 release and then following up with the empty value changes in 4.0.

Reply:
Makes sense. I've been using the `return self.empty_value` pattern for a while and I think it should be changed everywhere because existing fields are often used a reference when new fields are added. This can happen in another PR.